src/coreclr/jit/llvm.cpp

```diff
@@ -42,6 +42,25 @@ struct SpilledExpressionEntry
     CorInfoType m_CorInfoType;
 };
 
+enum LocalVarLocation
+{
+    LlvmStack,
+    ShadowStack
+};
+
+// local vars that don't need to be on the shadow stack
+struct LocalVar
+{
+    Value* llvmValue;
+    LocalVarLocation location;
+
+    LocalVar(LocalVarLocation location, Value* llvmValue) : llvmValue(llvmValue), location(location) {}
+    virtual Value* getValue(llvm::IRBuilder<>& builder)
```

Comment:
```suggestion
    Value* getValue(llvm::IRBuilder<>& builder)
```
No need for `virtual` anymore?

Reply:
thanks, missed that.